corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
@nickpell I offlined with Cory this morning, and edited https://gist.github.com/dannyroberts/460abd70be3d271fbc88 to account for this which I'd overlooked before (see the bold "is in fact critical" towards the bottom). Let's take to slack, but the solution here is

- Make sure any related document class *doesn't* subclass `CachedCouchDocumentMixin` if you're removing caching of one of its views
- Make sure to replace any caching you're removing with `QuickCachedDocumentMixin` + `quickcache`